Model/review.model.js

```diff
@@ -1,5 +1,27 @@
 const db = require("../db/connection.js");
 
+exports.selectReviews = (query) => {
+  // ONE QUERY ASH
+  const columnName = Object.keys(query)[0];
+  const whereValue = query[columnName];
+  let queryStr =
+    "SELECT reviews.*, COUNT(comment_id) AS comment_count FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id";
+  if (columnName) {
+    queryStr += ` WHERE ${columnName} = $1`;
+  }
+  queryStr += " GROUP BY reviews.review_id;";
+
+  if (whereValue) {
+    return db.query(queryStr, [whereValue]).then((results) => {
+      return results.rows;
+    });
+  } else {
+    return db.query(queryStr).then((results) => {
+      return results.rows;
+    });
+  }
+};
```

Comment:
This is good, but we're missing a certain situation.  What if there are no reviews found, but that category does exist (in which case you want to let your users know that there will be content found there later, it just hasn't been made yet)?  Currently this would be treated the same as the category being typed in incorrectly.

So if we receive an empty array back from our query, we have to find out if the category does exist in our database, and then change the response we send accordingly.

Reply:
Ah I see well I'll get that sorted before the next Pull Request